packages/fast-components-react-base/src/context-menu-item/context-menu-item.tsx

```diff
@@ -87,30 +87,30 @@ class ContextMenuItem extends Foundation<
      * Handle the keydown event of the item
      */
     private handleMenuItemKeyDown = (e: React.KeyboardEvent<HTMLDivElement>): void => {
-        if (this.props.disabled) {
-            return;
-        }
-
-        if (typeof this.props.onInvoke === "function") {
+        if (typeof this.props.onInvoke === "function" && !this.props.disabled) {
             switch (e.keyCode) {
                 case KeyCodes.enter:
                 case KeyCodes.space:
                     this.props.onInvoke(e, this.props);
                     break;
             }
         }
+
+        if (typeof this.props.onKeyDown === "function") {

```

Comment:
Do we want to execute this if the item is disabled?

Reply:
yes - this should work as an unhandledProp.